Server/painttheworld/game.py

```diff
@@ -101,9 +101,14 @@ def add_user(self, lat, lon):
         else:
             return -1
 
+<<<<<<< HEAD
```

Comment:
I'll get the master versions of server stuff.  It's because this branch was based on a previous version of master.